administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -689,7 +785,7 @@ public function finaliseUpgrade()
 
         try {
             $db->execute();
-        } catch (\RuntimeException $e) {
+        } catch (RuntimeException $e) {
```

Comment:
```suggestion
        } catch (\RuntimeException $e) {
```
Revert this change because in J4 the file is namespaced.